pyInterface/module/_fit.py

```diff
@@ -131,4 +139,71 @@ def pwaNloptFit(eventAndAmpFileDict,
 		                                       saveSpace          = saveSpace,
 		                                       verbose            = verbose)
 		fitResults.append(fitResult)
+		iFitResult = len(fitResults)-1
```

Comment:
It would be nice to also have this also in the pwaFit() function. pwaFit() and pwaNloptFit() actually are very similar and seem to differ only in the call to the fitter. So maybe one could move some of the shared code to service functions.